set-quotas.py

```diff
@@ -202,9 +203,18 @@ def match_keystone_project(all_ks_projects, form_project):
                               quota_list=quota_list,
                               **quota_cfg)
         msg.send()
+        copy_index.append(project['row'])
         print "Successfully updated quotas for project {}".format(
               ks_project.name)
 
+    # FIXME: code is duplicated from addusers.py, centralize it
+    if copy_index:
+        copy_rows = [r for r in rows if rows.index(r) in copy_index]
+        sheet.append_rows(copy_rows, target="Processed Requests")
+        result = sheet.delete_rows(copy_index, 'Form Responses 1')
```

Comment:
just for information, the deletes actually happen in reverse order (this is handled by spreadsheet.py).  Otherwise, if you delete row 0, then what used to be row 2 is now row 1, and then when you delete row 2 you delete the wrong row.